INSTALL.md

```diff
@@ -77,3 +77,14 @@ Please see [INSTALL.make.md](dev/doc/INSTALL.make.md) for build and
 installation instructions using `make`. If you wish to experiment with
 the Dune-based system see the [dune guide for
 developers](dev/doc/build-system.dune.md).
+
+Notes on build-time dependencies
+--------------------------------
+
+- The OCaml compiler and findlib are build-time dependencies, but also
+  run-time dependencies if you wish to use the native compiler.
+
+- When loading plugins, you should make sure that these were compiled
+  with the same version of OCaml as Coq.  This is also true when
```

Comment:
I'd write "same OCaml setup" instead of version. In particular, all libraries and transitive dependencies have to match.